src/lib.rs

```diff
@@ -1435,20 +1428,20 @@ impl Ssl {
         }
     }
 
-    fn load_verify_certs(ctx: &SslContext) -> Result<RootCertStore, error::Error> {
-        let mut verify_roots = ctx.verify_roots.clone();
-
+    fn load_verify_certs(ctx: &SslContext) -> Result<x509::OwnedX509Store, error::Error> {
         // If verify_roots isn't empty then it was configured with `SSL_CTX_load_verify_file`
         // or `SSL_CTX_load_verify_dir` and we should use it as-is.
-        if !ctx.verify_roots.is_empty() {
-            return Ok(verify_roots);
+        if !ctx.verify_x509_store.is_empty() {
+            return Ok(ctx.verify_x509_store.clone());
         }
 
         // Otherwise, try to load the default cert file or cert dir.
+        let mut verify_roots = x509::OwnedX509Store::default();
+
         if let Some(default_cert_file) = &ctx.default_cert_file {
-            verify_roots.add_parsable_certificates(x509::load_certs(
+            verify_roots.add(x509::load_certs(
```

Comment:
Maybe add an `OwnedX509Store` method that abstracts over the `add(x509::load_certs(_))` pattern?